packages/babel-parser/test/fixtures/typescript/types/tuple-keyword-labeled-babel-7/input.ts

```diff
@@ -0,0 +1,4 @@
+type FuncWithDescriptionBabel7 = [
+  function: (...args: any[]) => any,
+  string: string
```

Comment:
Can you augment the test case to include optional keyword labeled tuple? such as `void?: string`